README.md

```diff
@@ -226,3 +230,25 @@ ln -s ../../../web/sites/simpletest repos/drupal/sites
 Drupal's /composer folder is not symlinked and therefore isn't visible to
 Composer. It's needed for some tests, and so is declared as an autoload
 location.
+
+## Core Development Using DDEV
+
+1. Clone this repository by using `git clone --branch=main https://github.com/joachim-n/drupal-core-development-project.git`
+2. cd repos
+3. git clone --branch=10.1.x https://git.drupalcode.org/project/drupal.git drupal
+4. ddev get drud/ddev-selenium-standalone-chrome (downloads latest chrome driver)
+5. ddev composer install (from project root)
+6. ddev composer require drupal/admin_toolbar drupal/devel
+7. cd repos/drupal ; `git status` to track core changes
```

Comment:
I don't think this is the right approach at all. Instead, 

```
mkdir drupal-core-development-project && cd drupal-core-development-project
ddev composer create -y --no-install joachim-n/drupal-core-development-project
ddev composer install
ddev get drud/ddev-selenium-standalone-chrome
```

This will make it far more like ordinary DDEV project management, like the quickstarts in [docs](http://127.0.0.1:8000/en/latest/users/quickstart/#drupal)

Reply:
Hi @rfay , I tried above instructions and observed below .

1. `ddev composer create -y --no-install joachim-n/drupal-core-development-project` is not creating project it is asking to initialise a project by running `ddev config` which is a additional step.

> could not find a project in /Users/user1/docker/drupal-core-development-project. Have you run 'ddev config'? Please specify a project name or change directories: no .ddev/config.yaml file was found in this directory or any parent

2. It asks for `Docroot Location (current directory):` which user wont be aware of.
3. You can not specify core version to download from repo
4. It is taking atleast 2x time to setup project if you are lucky enough it get finished .
5. At last there is a message stating `Moving install to composer root` which is taking forever to complete.
6. It gives a bad experience to the user.

So IMHO current instructions in README servers better.

Reply:
You have to create a directory and `cd` into the directory and `ddev config`. This is standard stuff, maybe you can experiment with the [quickstarts](https://ddev.readthedocs.io/en/latest/users/quickstart/#drupal) and see how they work, pattern it on them. Normally for Drupal, `ddev config --docroot=web --project-type=drupal10 --create-docroot`

Current instructions are not normal DDEV stuff and use composer on the host, which may be a different PHP version etc. Please try again.

Reply:
Ok, do let me know if you have any particular steps that you would like to have in particular manner in readme, if you wish to update it I am more than happy. Thanks for all your review comments so far.